From 757dd6cbca4e987222743e07a18e0903748083ad Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 24 Mar 2022 10:34:10 +0200
Subject: [PATCH] [SW-69581] gaudi3: pll: adjust ref clk for pldm

PLL reference clk in PLDM emulator might change for different HW
images. This patch sets the clock value for pldm HW image 2.03

Change-Id: I1fc631809814f9ecc8f7b42150ce9b26c69421c8
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/pldm.c b/arch/gaudi3/arc/pldm.c
index 3f73cfe..eda3259 100644
--- a/arch/gaudi3/arc/pldm.c
+++ b/arch/gaudi3/arc/pldm.c
@@ -11,9 +11,9 @@
 #include <pll.h>
 #include "hl.h"
 
-#define SYS_CLOCK_FREQUENCY	155000ULL
+#define SYS_CLOCK_FREQUENCY	21300ULL /* actual freq of clk_250mhz */
 
-#define UART_CLOCK_FREQUENCY	11900ULL
+#define UART_CLOCK_FREQUENCY	SYS_CLOCK_FREQUENCY
 #define PLDM_UART_BAUDRATE	300
 
 u64 hl_pll_timestamp_freq_get(void)
